src/instructlab/configuration.py

```diff
@@ -740,8 +763,6 @@ class Config(BaseModel):
     general: _general = Field(
         default_factory=_general, description="General configuration section."
     )
-    # model configuration
-    model_config = ConfigDict(extra="ignore")
```

Comment:
I think this is a pydantic thing not a model thing, I could be wrong. I think other classes have it

Reply:
Yes good catch